src/components/policy/policy_regular/include/policy/policy_manager_impl.h

```diff
@@ -819,6 +821,17 @@ class PolicyManagerImpl : public PolicyManager {
       const std::shared_ptr<policy_table::Table> update,
       const std::shared_ptr<policy_table::Table> snapshot);
 
+  /**
+   * @brief Compares current policies to the updated one.
+   * Trigger actions in case if certain fields after update were changes.
```

Comment:
`changes` -> `changed`

Reply:
@jacobkeeler Thanks. Fixed in 8ef7e5cb3072a4010f68ef0c440df7339be98e01